pennylane_cirq/simulator_device.py

```diff
@@ -188,6 +188,14 @@ def generate_samples(self):
             [self._result.measurements[str(wire)].flatten() for wire in range(self.num_wires)]
         ).T.astype(int)
 
+    def expval(self, observable):
+        # pylint: disable=missing-function-docstring
+        if not self.analytic or not hasattr(self._simulator, "simulate_expectation_values"):
+            return super().expval(observable)
+        return self._simulator.simulate_expectation_values(
+            self.circuit, self.to_paulistring(observable)
+        )[0]
```

Comment:
Nice, I suppose this leads to a performance improvement?

Reply:
Yes, it essentially moves the responsibility of calculating the expectation value to the cirq device rather than just pennylane. The downstream device can then optimize how it's calculated.

Reply:
Awesome. A lot of PennyLane plugin patterns are repeated from a time (a long way!) back when other frameworks didn't provide (accessible) expectation APIs. Now that this is commonplace, we should take this into account more when writing plugins.

Reply:
Agreed, especially for more future remote services (both simulation and real QC).